pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -42,7 +42,7 @@ cd ${SUBJECT}/anat
 # NOTE: as the model for both ventral and dorsal rootlets is not part of SCT yet, we run directly the nnUNet model using
 # the wrapper script run_inference_single_subject.py from the model-spinal-rootlets repository
 # https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_ventral_rootlets/run_inference_single_subject.py
```

Comment:
```suggestion
# https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_ventral_rootlets/run_inference_single_subject.py
# Note: we use SCT python because it has nnUNet installed 
```